lib/ffi/clang/cursor.rb

```diff
@@ -62,6 +62,14 @@ def completion
 				CodeCompletion::String.new Lib.get_cursor_completion_string(@cursor)
 			end
 
+			def anonymous?
+				Lib.cursor_is_anonymous(@cursor) != 0
+			end
+
+			def anonymous_record_decl?
```

Comment:
Is there a difference between `decl` and `declaration`?

As a general rule of thumb I try to avoid abbreviations.

Reply:
Sounds good to me - updated.